source/geometry_model/spherical_shell.cc

```diff
@@ -76,39 +76,14 @@ namespace aspect
           Assert (false, ExcInternalError());
         }
 
-      // Use a manifold description for all cells. use manifold_id 99 in order
-      // not to step on the boundary indicators used below
-      static const SphericalManifold<dim> spherical_manifold;
-      coarse_grid.set_manifold (99, spherical_manifold);
+      set_manifold_ids(coarse_grid);
```

Comment:
add comment:
"First set manifold ids for everything (interior+boundary), then reset the boundary, because the manifold has no normals:"